readme.mdown

```diff
@@ -0,0 +1,62 @@
+# Scrivito Portal App
+
+[![CMS: Scrivito](https://img.shields.io/badge/CMS-Scrivito-brightgreen.svg)](https://scrivito.com) [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT) [![CircleCI](https://dl.circleci.com/status-badge/img/gh/Scrivito/scrivito-portal-app/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/Scrivito/scrivito-portal-app/tree/main) [![GitHub last commit](https://img.shields.io/github/last-commit/Scrivito/scrivito-portal-app.svg)](https://github.com/Scrivito/scrivito-portal-app) [![GitHub top language](https://img.shields.io/github/languages/top/Scrivito/scrivito-portal-app.svg)](https://github.com/Scrivito/scrivito-portal-app)
+
+[Scrivito](https://www.scrivito.com/?utm_source=github&utm_medium=natural&utm_campaign=github_portal_app) is the world's most secure, serverless JavaScript CMS. Based on ReactJS, it is completely maintenance-free, cost-effective, and has unprecedented security. Learn more about Scrivito at [scrivito.com](https://www.scrivito.com/?utm_source=github&utm_medium=natural&utm_campaign=github_portal_app).
+
+This portal app shows Scrivito’s features and is a great basis for your own Scrivito website projects.
```

Comment:
* [ ] FYI In our internal communication we always treat "Portal App" as a name, with uppercase initials. How about: "This portal app" -> "The Portal App" or -> "The Scrivito Portal App"?